src/WorksomeSniff/Sniffs/Classes/DisallowPhpUnitTestSniff.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Worksome\WorksomeSniff\Sniffs\Classes;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class DisallowPhpUnitTestSniff implements Sniff
+{
+    public function register(): array
+    {
+        return [
+            T_DOC_COMMENT_TAG,
+            T_FUNCTION,
+        ];
+    }
+
+    public function process(File $phpcsFile, $stackPtr): void
+    {
+        // Make sure the file is in the `tests` directory.
+        if (! str_contains($phpcsFile->getFilename(), DIRECTORY_SEPARATOR . 'tests' . DIRECTORY_SEPARATOR)) {
+            return;
+        }
+
+        match($phpcsFile->getTokens()[$stackPtr]['code']) {
```

Comment:
There is a method to get the code for a specific pointer, consider using that so we don't depend on the full array of all tokens

Reply:
Can you remember what it is? I've had a look through and can't find anything specific

Reply:
`getTokensAsString` is the method 👍